.github/workflows/accelerated-testing.yml

```
@@ -0,0 +1,139 @@
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v1
+      with:
+        file: ./coverage.xml
+        flags: ${{ matrix.test-group }}
+        name: ${{ matrix.test-group }}-py${{ matrix.python-version }}
+
+  security-scan:
+    needs: parallel-testing
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)